lib/moab/storage_object_validator.rb

```diff
@@ -0,0 +1,198 @@
+require 'moab'
+
+# Shameless gree: repetitious code included.
```

Comment:
I know I told you to put this here, but given that you have a class comment ... perhaps it's part of the class comment, not the module comment?